src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -55,8 +54,8 @@ internal class SqlServerSearchService : SearchService
         private const string SortValueColumnName = "SortValue";
         private readonly SchemaInformation _schemaInformation;
         private readonly RequestContextAccessor<IFhirRequestContext> _requestContextAccessor;
-        private readonly ISearchParameterDefinitionManager.SearchableSearchParameterDefinitionManagerResolver _searchParameterDefinitionManagerResolver;
         private const int _resourceTableColumnCount = 10;
+        private readonly SearchParameterInfo _fakeLastUpdate = new SearchParameterInfo(SearchParameterNames.LastUpdated, SearchParameterNames.LastUpdated);
```

Comment:
>_fakeLastUpdate [](http://example.com/codeflow?start=45&length=15)

nit: should we use a different name?

Reply:
I'll be happy to... 
Do you have anything in mind?